packages/app/models/stack/types.ts

```diff
@@ -0,0 +1,6 @@
+export enum FREQUENCY_OPTIONS {
+  hour = "hour",
+  day = "day",
+  week = "week",
+  month = "month",
+}
```

Comment:
is this a type? 

I see types as typescript things: types, interfaces, props, etc.

 I think it belong to the `Stack` model

Reply:
`enum` can be used for types. The main difference I'd point out is if you want to allow the dev to hardcode text values or to use the `enum` as single source of truth.

So these two should yield the same results (type-wise speaking):
`type FrequencyOptions = "hour" | "day" | "week" | "month";`

But the way to pass down these values changes. If you type a value as `FrequencyOptions`, then you can pass down `"hour"` as string and it will pass.

If you type a value as `FREQUENCY_OPTIONS`, then you HAVE to pass down `FREQUENCY_OPTIONS.hour` as the value, because if you try to pass down `"hour"` string value it will fail.

----------------------

I like enums better when it's a specific set of string/number values, as you always use the same source of truth and if you have to update the source used for type, updating the `enum` will impact everywhere where it's being used, whereas updating the `type` will also imply to update all the places where it's being used.

Reply:
As Rorry said, it can be used for types (and I defined this way to use it as a type), so I would leave it here.

Reply:
Thanks for sharing your thoughts @ElRodrigote.  

I agree with you guys.